RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -58,7 +58,7 @@ PixelThresholdClusterizer::PixelThresholdClusterizer
     thePhase2KinkADC( conf.getParameter<int>("Phase2KinkADC") ),
     theNumOfRows(0), theNumOfCols(0), theDetid(0),
     // Get the constants for the miss-calibration studies
-    doMissCalibrate( conf.getUntrackedParameter<bool>("MissCalibrate",true) ),
+    doMissCalibrate( conf.getParameter<bool>("MissCalibrate") ),
```

Comment:
Please add the 'true' value here.
In fact, all fillDescriptions parameters, wehther untracked or not,
should have a corresponding default value so that it
can be inserted in configs not containing the parameter.
This way it does not need to be added via the HLT customizer.
(Though in this case it may be needed because it is changing
untracked to tracke, but then the HLT customizer should
simply delete the untracked parameter, whil fillDescriptions
then injects the tracked parameter with its associated value)